app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide :title, "Help" %>
```

Comment:
Chú tìm hiểu I18n cho toàn bộ text hiển thị trên html

em self check và fix tương tự cho các file html nhé